pkg/api/ingest_collector.go

```diff
@@ -18,6 +18,7 @@
 package api
 
 type IngestCollector struct {
-	HostName string `yaml:"hostName" doc:"the hostname to listen on"`
-	Port     int    `yaml:"port" doc:"the port number to listen on"`
+	HostName    string `yaml:"hostName" doc:"the hostname to listen on"`
+	Port        int    `yaml:"port" doc:"the port number to listen on"`
+	BatchMaxLen int    `yaml:"batchMaxLen" doc:"the number of accumulated flows before being forwarded for processing"`
```

Comment:
@eranra @mariomac Do we also want to be able to specify a timeout value after which to send whatever we have?

Reply:
@mariomac I like the one-by-one + channels approach ... getting more performance by "multi-thread/core" is what we need.